fiftyone.devicedetection.onpremise/deviceDetectionOnPremisePipelineBuilder.js

```diff
@@ -102,6 +104,7 @@ class DeviceDetectionOnPremisePipelineBuilder extends PipelineBuilder {
       licenceKeys = null,
       dataFile = null,
       autoUpdate = true,
+      dataFileUpdateBaseUrl = constants.dataFileUpdateBaseUrl,
```

Comment:
let's maybe pass `null` as a default here since we added a check for null anyway.